repl/src/main/scala/org/apache/spark/repl/ExecutorClassLoader.scala

```diff
@@ -55,6 +57,14 @@ class ExecutorClassLoader(conf: SparkConf, classUri: String, parent: ClassLoader
     }
   }
 
+  override def getResource(name: String): URL = {
+    parentLoader.getResource(name)
```

Comment:
> I'm wondering it is safe to return URL from findResource() and findResources() which doesn't point to local file

That should be perfectly fine. That's how URLClassLoader works, after all. The only potential odd thing would be `getResourceAsStream`, which returns an `InputStream`, and my guess is if the JDK's `URL` class doesn't support the protocol, would cause an error unless you overrode it here.

So perhaps it's too much to worry about and we can just assume that no one will do that, and fix it if someone ever needs that feature.